### ISSUE TITLE
Enabled bluetooth every time I boot > Need to deactivate it every time

### ISSUE THREAD

**Issue by MartinX3** [systemd/systemd#7304]
### Submission type

 - Request for enhancement (RFE)

### systemd version the issue has been seen with

> …systemd 229-4ubuntu4 

### Used distribution

> … Linux Mint 18.2 alias Ubuntu 16.04

I want an option that bluetooth is disabled by default.
The Linux Mint Master Developer told me, that I need to make a bug report at systemd.
https://github.com/linuxmint/blueberry/issues/42#issuecomment-343212154

**Comment by boucman** [comment]
that's more a support request than a feature request, and it's for an old, unsupported version.

That being said, i'm not even sure it's a systemd problem. systemd can be configured to not stop bluez at startup, but what you want is to tell bluez to not activate the bluetooth, which is a bluez configuration problem, probably...

**Comment by mbiebl** [comment]
While systemd provides bluetooth.target, it doesn't actually provide the bluetooth service itself, so it's not really a systemd issue.

If you simply want to prevent the bluetooth service from being started, you can consider masking the service. It's not clear if that is what you want realy. But this is not a user support forum but a bug tracker for systemd, so closing this issue.

**Comment by MartinX3** [comment]
I only want an option, that Bluetooth is deactivated by default.
I don't want to deactivate it every time, I boot my PC.

So I thought it was an ISSUE in the Bluetooth application, but they told me, that it is a SystemD issue.
(https://github.com/linuxmint/blueberry/issues/42#issuecomment-343212154)
What should I do now?
Who is the recipient for my issue? :(

**Comment by boucman** [comment]
choosing if a service is turned on or off by default is really a distro decision. systemd provides the tools for the distro to choose, but it's really their choice.
I really believe it's a linuxmint feature request (not a bug, it's a distro choice)

**Comment by poettering** [comment]
what do you actually mean by "bluetooth"? the rfkill LED or so? or the bluetooth service? Have you tried disabling it with "systemctl disable bluetooth"? or even with "systemctl mask bluetooth"?

**Comment by MartinX3** [comment]
In the application "blueberry" in linux mint, I can disable the bluetooth in the GUI.
But I don't want to do it every time by hand, when I boot my system.
So I wrote an issue there (linuxmint/blueberry#42 (comment)) to enhance the software, to give the possibility, to let me choose, to disable the bluetooth by default.
But they told me, that this is a SystemD issue and closed my issue.

So I reported it here.
I'm a little bit confused who is responsible now for this issue?

To write my own bash script, which is runned on every boot isn't a real opinion for me, if there should be an option in the bluetooth manager application gui.
